activities-examples/src/test/workflow.test.ts

```diff
@@ -67,7 +71,11 @@ describe('example workflow', function () {
 
     await assert.rejects(
       () => workflow.execute(),
-      (err: any) => err.name === 'WorkflowExecutionFailedError' && err.cause.cause.message === 'example error'
+      (err: unknown) =>
+        err instanceof WorkflowExecutionFailedError &&
+        err.cause &&
+        'cause' in err.cause &&
+        err.cause['cause']['message'] === 'example error'
```

Comment:
If we want to be exact use something like this:
`err.cause instanceof ActivityFailure && err.cause.cause && intstanceof ApplicationFailure && err.cause.cause.message === 'example error'`